scalafmt-dynamic/src/test/scala/org/scalafmt/dynamic/ScalafmtVersionSuite.scala

```diff
@@ -22,28 +22,36 @@ class ScalafmtVersionSuite extends FunSuite {
     )
     assertEquals(
       ScalafmtVersion.parse("2.2.3-SNAPSHOT"),
-      Some(ScalafmtVersion(2, 2, 3, 0, "-SNAPSHOT"))
+      Some(ScalafmtVersion(2, 2, 3, 0, "", "-SNAPSHOT"))
     )
     assertEquals(
       ScalafmtVersion.parse("2.0.0-RC1-SNAPSHOT"),
-      Some(ScalafmtVersion(2, 0, 0, 1, "-SNAPSHOT"))
+      Some(ScalafmtVersion(2, 0, 0, 1, "", "-SNAPSHOT"))
     )
     assertEquals(
       ScalafmtVersion.parse("2.2.2-SNAPSHOT"),
-      Some(ScalafmtVersion(2, 2, 2, 0, "-SNAPSHOT"))
+      Some(ScalafmtVersion(2, 2, 2, 0, "", "-SNAPSHOT"))
     )
     assertEquals(
       ScalafmtVersion.parse("99.99.99-RC99-SNAPSHOT"),
-      Some(ScalafmtVersion(99, 99, 99, 99, "-SNAPSHOT"))
+      Some(ScalafmtVersion(99, 99, 99, 99, "", "-SNAPSHOT"))
     )
     assertEquals(
       ScalafmtVersion.parse("99.99.99-RC99+foobar"),
-      Some(ScalafmtVersion(99, 99, 99, 99, "+foobar"))
+      Some(ScalafmtVersion(99, 99, 99, 99, "+foobar", ""))
+    )
+    assertEquals(
+      ScalafmtVersion.parse("99.99.99+foobar"),
```

Comment:
is this a duplicate test? (see next one)

Reply:
Oh, right, I'll remove one :+1: